src/style.css

```diff
@@ -1 +1,85 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: black;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  background-color: white;
+}
+
+.header--inner {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.header__logo--inner {
+  /*as i got it, due to the logo--link height of logo--inner becomes 40+ px*/
+  height: 40px;
+}
+
+.header__logo--img {
+  position: relative;
+  top: 50%;
+  transform: translateY(-25%);
+}
+
+.header__navigation {
+  /* padding: 23px 0; */
+  display: flex;
+  align-items: center;
+}
+
+.header__nav--list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.header__items:not(:last-child) {
+  margin-right: 20px;
+}
+
+.header__link {
+  text-decoration: none;
+  color: inherit;
+  text-transform: uppercase;
+  padding: 5px 0;
+  position: relative;
+
+  /* only for +2 tests*/
+  line-height: 60px;
+}
+
+.header__link:hover {
+  color: #00acdc;
+}
+
+.is-active:focus {
+  color: #00acdc;
+}
+
+.is-active:focus::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 3px;
+  background-color: #00acdc;
+  left: 0;
+  bottom: -18.5px;
```

Comment:
it should be `bottom: 0`, if it does not work it means that you made mistake in previous styles